packages/components/src/number-control/stories/index.tsx

```diff
@@ -62,3 +62,28 @@ export const Default = Template.bind( {} );
 Default.args = {
 	label: 'Value',
 };
+
+// Check if this was broken and at what point
+// in particular on commit actions, when `min` is not `undefined`
+export const Uncontrolled: ComponentStory< typeof NumberControl > = ( {
```

Comment:
This Storybook example can be removed before merging — it's mostly here to help manual testing with the uncontrolled version of the component